keyboards/fancytech/fancyalice66/info.json

```diff
@@ -0,0 +1,249 @@
+            {"flags": 4, "matrix": [2, 2], "x": 30, "y": 32},
+            {"flags": 4, "matrix": [2, 3], "x": 45, "y": 32},
+            {"flags": 4, "matrix": [2, 4], "x": 60, "y": 32},
+            {"flags": 4, "matrix": [2, 5], "x": 75, "y": 32},
+            {"flags": 4, "matrix": [2, 7], "x": 105, "y": 32},
+            {"flags": 4, "matrix": [2, 8], "x": 120, "y": 32},
+            {"flags": 4, "matrix": [2, 9], "x": 135, "y": 32},
+            {"flags": 4, "matrix": [2, 10], "x": 150, "y": 32},
+            {"flags": 4, "matrix": [2, 11], "x": 164, "y": 32},
+            {"flags": 4, "matrix": [2, 12], "x": 180, "y": 32},
+            {"flags": 1, "matrix": [2, 13], "x": 194, "y": 32},
+            {"flags": 4, "matrix": [2, 15], "x": 224, "y": 32},
+
+            {"flags": 1, "matrix": [3, 0], "x": 0, "y": 48},
+            {"flags": 4, "matrix": [3, 1], "x": 15, "y": 48},
+            {"flags": 4, "matrix": [3, 2], "x": 30, "y": 48},
+            {"flags": 4, "matrix": [3, 3], "x": 45, "y": 48},
+            {"flags": 4, "matrix": [3, 4], "x": 60, "y": 48},
+            {"flags": 4, "matrix": [3, 5], "x": 75, "y": 48},
+            {"flags": 4, "matrix": [3, 7], "x": 105, "y": 48},
+            {"flags": 4, "matrix": [3, 8], "x": 120, "y": 48},
+            {"flags": 4, "matrix": [3, 9], "x": 135, "y": 48},
+            {"flags": 4, "matrix": [3, 10], "x": 150, "y": 48},
+            {"flags": 4, "matrix": [3, 11], "x": 164, "y": 48},
+            {"flags": 4, "matrix": [3, 12], "x": 180, "y": 48},
+            {"flags": 4, "matrix": [3, 14], "x": 209, "y": 48},
+
+            {"flags": 1, "matrix": [4, 0], "x": 0, "y": 64},
+            {"flags": 1, "matrix": [4, 2], "x": 30, "y": 64},
+            {"flags": 4, "matrix": [4, 3], "x": 45, "y": 64},
+            {"flags": 1, "matrix": [4, 6], "x": 90, "y": 64},
+            {"flags": 4, "matrix": [4, 7], "x": 105, "y": 64},
+            {"flags": 1, "matrix": [4, 10], "x": 150, "y": 64},
+            {"flags": 4, "matrix": [4, 13], "x": 194, "y": 64},
+            {"flags": 4, "matrix": [4, 14], "x": 209, "y": 64},
+            {"flags": 4, "matrix": [4, 15], "x": 224, "y": 64},
+
+            {"flags": 2, "x": 37, "y": 64},
+            {"flags": 2, "x": 75, "y": 64},
+            {"flags": 2, "x": 112, "y": 64},
+            {"flags": 2, "x": 149, "y": 64},
+            {"flags": 2, "x": 187, "y": 64},
+            {"flags": 2, "x": 224, "y": 64},
+            {"flags": 2, "x": 224, "y": 32},
+            {"flags": 2, "x": 224, "y": 0},
+            {"flags": 2, "x": 187, "y": 0},
+            {"flags": 2, "x": 149, "y": 0},
+            {"flags": 2, "x": 112, "y": 0},
+            {"flags": 2, "x": 75, "y": 0},
+            {"flags": 2, "x": 37, "y": 0},
+            {"flags": 2, "x": 0, "y": 0},
+            {"flags": 2, "x": 0, "y": 32},
+            {"flags": 2, "x": 0, "y": 64}
+        ],
+        "max_brightness": 200,
+        "rgblight.pin": "B0"
```

Comment:
```suggestion
        "pin": "B0"
```

Reply:
Resolved in [
16454d7](https://github.com/qmk/qmk_firmware/pull/20647/commits/16454d7897942d03de24058c14cf2cfe86efe766). Rest of commands follow suit.